### PR TITLE
distributed: init at 1.10.0 (+init and update dependencies)

### PR DESCRIPTION
**Pull request by teh**
I couldn't get nox-review to build only the pythonPackages, it's trying to build hundreds of changes. I manually built and ran on pythonPackages, python3Packages and python35Packages.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -4609,6 +4609,98 @@ in modules // {
...
+    };
+
+    buildInputs = with self; [ pytest docutils ];
+    propagatedBuildInputs = with self; [
+      dask six boto3 s3fs tblib locket msgpack click cloudpickle tornado
+      psutil botocore
+    ] ++ (if !isPy3k then [ singledispatch ] else []);
+
+    # AttributeError: module 'pytest' has no attribute 'config'
+    doCheck = false;
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: likely you need to run pytest yourself
```
checkPhase = ''
  py.test
''

**Comment by FRidh** [comment]
could you divide it into a commit per package. Thanks.

File: pkgs/top-level/python-packages.nix (hunk 1 above)
Comment (teh) [inline pkgs/top-level/python-packages.nix]: I tried the invocation they're using for travis but that still doesn't work `ValueError: no option named '--runslow'` 

This package also seems to need a successful install before running the tests (i.e. python setup.py install has to install binaries into some PATH accessible location).

I spent a fair while on trying to get tests to run but not sure it's easy.

**Comment by teh** [comment]
done, ptal

**Pull request merged**